build-browser.js

```diff
@@ -0,0 +1,23 @@
+const pkg = require('./package.json');
+const rollup = require('rollup').rollup;
+const babel = require('rollup-plugin-babel');
+
+rollup({
+  entry: 'src/index.js',
+  plugins: [
+    babel({
+      presets: ['es2015-rollup'],
+      plugins: ['transform-node-env-inline'],
+      babelrc: false,
+    }),
+  ],
+}).then(bundle => (
+  bundle.write({
+    dest: 'validator.js',
+    format: 'umd',
+    moduleName: pkg.name,
```

Comment:
@chriso I do not use Ember so I'm not sure. Maybe some one can help test it? If it does not work I can open a rollup issue to fix it there.